### ISSUE TITLE
Update Windows build instructions or release updated binaries for Windows

### ISSUE THREAD

**Issue by HwapX** [midori-browser/core#209]
I'am wanting to use Midori on Windows but the binaries available here(https://www.midori-browser.org/) are outdated.

I tried to cross compile on Fedora 29(container) to Windows but i cant find some packages.

```
-- The C compiler identification is GNU 8.2.0
-- Check for working C compiler: /usr/bin/i686-w64-mingw32-gcc
-- Check for working C compiler: /usr/bin/i686-w64-mingw32-gcc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- valac 0.42.3 found
-- Found PkgConfig: /usr/bin/i686-w64-mingw32-pkg-config (found version "0.28") 
-- Checking for modules 'sqlite3>=3.6.19;gio-2.0>=2.48.0;libsoup-2.4>=2.48.0'
--   Found sqlite3, version 3.24.0
--   Found gio-2.0, version 2.58.1
--   Found libsoup-2.4, version 2.59.90.1
-- Checking for modules 'gtk+-3.0>=3.12.0;webkit2gtk-4.0>=2.16.6;gcr-ui-3>=2.32;libpeas-gtk-1.0'
--   No package 'webkit2gtk-4.0' found
--   No package 'gcr-ui-3' found
--   No package 'libpeas-gtk-1.0' found
CMake Error at /usr/share/cmake/Modules/FindPkgConfig.cmake:436 (message):
  A required package was not found
Call Stack (most recent call first):
  /usr/share/cmake/Modules/FindPkgConfig.cmake:602 (_pkg_check_modules_internal)
  CMakeLists.txt:52 (pkg_check_modules)


-- Configuring incomplete, errors occurred!
See also "/share/c/midori/_mingw32/CMakeFiles/CMakeOutput.log".

```

**Comment by kalikiana** [comment]
That's basically why there isn't a new build yet. Someone needs to procure those binaries. WebKit2GTK+ in particular doesn't currently build on Windows at all.

**Comment by AFaustini** [comment]
The regular webkit builds well in windows https://build.webkit.org/builders/Apple%20Win%2010%20Release%20%28Build%29/builds/1244

**Comment by kalikiana** [comment]
> The regular webkit builds well in windows [build.webkit.org/builders/Apple%20Win%2010%20Release%20%28Build%29/builds/1244](https://build.webkit.org/builders/Apple%20Win%2010%20Release%20%28Build%29/builds/1244)

Yeah. There isn't a reason WebKitGTK+ shouldn't. It used to (Midori used it). But the problem here is that we need a version with GTK+ support.

**Comment by kalikiana** [comment]
Another option we have is https://github.com/tiliado/valacef which is to say CFE (Chromium for Embedded) which would seem to work on Windows. Although it's a self-proclaimed WIP and will add complexity to the build.

**Comment by Poncho09** [comment]
Exist anothers alternatives for Windows for example: Pywebkitgtk https://wiki.python.org/moin/PyWebkitGtk, PyGTK https://pygobject.readthedocs.io/en/latest/ and the option native for Windows using C++ https://webkit.org/webkit-on-windows/#installing-developer-tools

**Comment by kalikiana** [comment]
Here's pointers from someone looking into updating the mingw32 build: https://gist.github.com/baylej/d32be86616104569b29b which contains notes on the relevant code paths and upstream work. If we wanna continue to use WebKitGTK+ on Windows this would be the way to go.

**Comment by kiwi0fruit** [comment]
[Here](https://www.reddit.com/r/browsers/comments/a3ykh8/are_there_any_modern_webkitbased_browsers_for/) is stated that 

> Webkit2 for Windows doesn't exist at all

does this really mean that starting some WebKit version nobody had built WebKit on Windows?

**Comment by kiwi0fruit** [comment]
[Here](https://www.reddit.com/r/browsers/comments/a3ykh8/are_there_any_modern_webkitbased_browsers_for/) is stated that 

> Webkit2 for Windows doesn't exist at all

does this really mean that starting some WebKit version nobody had built WebKit on Windows?

**Comment by kiwi0fruit** [comment]
I guess no Window builds: https://github.com/msys2/MINGW-packages/issues/3484

**Comment by heydojo** [comment]
https://github.com/Microsoft/vcpkg
Might be a solution to this issue the project is having with builds for Windows.
It's cross platform and I've used it with success on a project elsewhere.

On a side note; it might be helpful to remove the cross compile information in readme.md due to the fact that it clearly doesn't work.
I was baited into wasting my afternoon trying to build it.
(Thanks BTW.)

**Comment by ZachBacon** [comment]
> https://github.com/Microsoft/vcpkg
> Might be a solution to this issue the project is having with builds for Windows.
> It's cross platform and I've used it with success on a project elsewhere.
> 
> On a side note; it might be helpful to remove the cross compile information in readme.md due to the fact that it clearly doesn't work.
> I was baited into wasting my afternoon trying to build it.
> (Thanks BTW.)

It's no solution, the issue is webkitgtk itself. it needs proper api changes for it to support windows again. Their main focus iirc is linux focused, it doesn't even support macOS anymore.
